src/frontend/ikoner/DokumentIkon.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+interface IDokumentIkon {
+    className?: string;
+    height?: number;
+    width?: number;
+}
+
+export const DokumentIkon: React.FC<IDokumentIkon> = ({ className, width = 48, height = 48 }) => {
+    return (
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className={className}
+            height={height}
+            width={width}
+            viewBox="0 0 24 24"
+        >
+            <g
+                stroke="#0067c5"
```

Comment:
Ser et par steder at [navfarger](https://design.nav.no/resources/colors/) blir hardkoda, men går kanskje ikke bruke de alle steder? :(

Reply:
Yes, you can import the javascript variables from the designsystem, so hardcoded colors are not needed anywhere :) `import navFarger from 'nav-frontend-core';`

Reply:
@ninakjek @daphneleebeek Thank you for letting me know about this👍